geoserver_publisher/index.js

```diff
@@ -121,6 +124,40 @@ async function checkIfCoverageStoresExist(rasterMetaInf) {
   }
 }
 
+/**
+ * Creates a time-enabled layer in the GeoServer for the given raster mosaic if
+ * not existing by GeoServer REST-API.
+ *
+ * @param {Object} rasterMetaInf
+ */
+async function createRasterTimeLayers (rasterMetaInf) {
+  const ws = rasterMetaInf.workspace;
+  const covStore = rasterMetaInf.coverage_store;
+  const srs = 'EPSG:3035'; //TODO check if defined in DB
+  // per convention coverage store name is layer name and layer title
+  const layerName = covStore;
+  const nativeName = rasterMetaInf.properties_path;
+  const layerTitle = covStore;
+
+  verboseLogging(`Checking existence for layer ${ws}:${layerName} in coverage store ${covStore} (native name: ${nativeName})`);
+
+  const layer = await grc.layers.get(covStore);
+
+  if (!layer) {
+    console.info(`Creating layer "${ws}:${layerName}" in store "${covStore}"`);
+    // publishDbRaster (workspace, coverageStore, nativeName, name, title, srs, enabled)
+    const layerCreated = await grc.layers.publishDbRaster(ws, covStore, nativeName, layerName, layerTitle, srs, true);
+    verboseLogging(`Layer "${ws}:${layerName}" created successfully?`, layerCreated);
+
+    console.info(`Enabling time for layer "${ws}:${layerName}"`);
+    const timeEnabled = await grc.layers.enableTimeCoverage(ws, covStore, layerName, 'DISCRETE_INTERVAL', 3600000, 'MAXIMUM');
+    verboseLogging(`Time dimension  for layer "${ws}:${layerName}" successfully enabled?`, timeEnabled);
```

Comment:
superfluous whitespace
```suggestion
    verboseLogging(`Time dimension for layer "${ws}:${layerName}" successfully enabled?`, timeEnabled);
```